app/models/claim/cleaner.rb

```diff
@@ -0,0 +1,46 @@
+class Claim
+  class Cleaner
```

Comment:
On reflection `Cleaner` feels a bit too generic. How about something more specific to communicate the behaviour more to the reader. Something like:

```ruby
Claim::PiiScrubber.scrub_completed_claims
```

We can then define "completed" as being rejected and paid claims that have not been modified in 2 months.